### DISCUSSION TITLE
Auto Suggest in PNP Search BOX

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#3750] [General]
Search box currently does not provide any suggestion after typing the search word, can it be possible to provide auto suggest option for it.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/36663570/f9a9289e-74e6-4c4a-a1a4-65d5f0eb740b)

I have already added extension from https://github.com/microsoft-search/pnp-modern-search-extensibility-samples.
Below is what my search webpart configuration shows

![image](https://github.com/microsoft-search/pnp-modern-search/assets/36663570/2c6b3c53-0213-4377-8ec7-515deaeb06f7)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/36663570/759cd333-03c8-4417-a8f4-4c76f3861ded)

**Comment by vishalgoyal2472** [comment]
I can see other issues which are posted after me addressed quickly but not mine.

please let me know if any other info needed?

**Reply by kasperbolarsen** [reply]
Questions are answered based on the responders knowledge.  I reply to a lot of questions,  but not if the topic is auto suggestions or extensions,  as I don't have any knowledge on those topics.
We can not provide any SLA as we are all volunteers,  but it is totally fair that you reach out, asking for help 😊

**Comment by vishalgoyal2472** [comment]
Hi 
Is there any solution for this?

**Reply by wobba** [reply]
See https://microsoft-search.github.io/pnp-modern-search/usage/search-box/#query-suggestions for how to enable suggestions from the static suggestions list.

**Reply by vishalgoyal2472** [reply]
Its already on but still not showing result.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/36663570/9005b950-226c-4f3b-ba75-e2f08b0fc582)

**Reply by wobba** [reply]
Have you uploaded static suggestions as per https://learn.microsoft.com/en-us/sharepoint/search/create-and-import-query-suggestions?
